sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/grafana.yaml

```diff
@@ -0,0 +1,100 @@
+    - id: grafana-api-token # Note: deprecated
+      rspecKey: S6762
+      metadata:
+        name: Grafana API tokens should not be disclosed
+      detection:
+        pre:
+          include:
+            content:
+              # This secret pattern checks for the prefix "eyJrIjoi" which is `{"k":"` B64 encoded.
+              # The chance that this can be found in another secret or even normal data is
+              # still somewhat likely, so to be sure we filter for "grafana" here.
+              - grafana
+        matching:
+          pattern: "\\b(eyJrIjoi[a-zA-Z0-9]+(?:={1,2}\\B|\\b))"
+      examples:
+        - text: |
+            curl 'https://example.grafana.net/api/dashboards/home' -X 'Authorization: Bearer eyJrIjoiWWpNc1ZPS5RjdjdHTWlyVmtTcjI0Vk9xN1RkOGI1V3YiLCJuIjoieZl5b19vb30vb23iLCJpZCI6MX0='
+          containsSecret: true
+          match: eyJrIjoiWWpNc1ZPS5RjdjdHTWlyVmtTcjI0Vk9xN1RkOGI1V3YiLCJuIjoieZl5b19vb30vb23iLCJpZCI6MX0=
+        - text:
+            | # Some random text I found using GitHub search; decoded it is `{"k":"@LoadText", "v":"Loading..."}`.
+            curl 'https://not-a-dashboard.sonarsource.com' -d 'eyJrIjoiQExvYWRUZXh0IiwgInYiOiJMb2FkaW5nLi4uIn0='
+          containsSecret: false
+    - id: grafana-service-account-token
+      rspecKey: S6762
+      metadata:
+        name: Grafana service account tokens should not be disclosed
+      detection:
+        pre:
+          include:
+            content:
+              - glsa_
+        matching:
+          pattern: "\\b(glsa_[a-zA-Z0-9]{32}_[a-f0-9]{8})\\b"
+      examples:
+        - text: |
+            import requests
+
+            token = 'glsa_geygSnIfuK5vBG0KgaflRCQfIb8mzaM7_b0999d91'  # Noncompliant
+            response = requests.get('https://example.grafana.net/api/dashboards/home', headers={
+                'Authorization': f'Bearer {token}',
+                'Content-Type': 'application/json'
+            })
+          containsSecret: true
+          match: glsa_geygSnIfuK5vBG0KgaflRCQfIb8mzaM7_b0999d91
+        - text: |
+            curl 'https://example.grafana.net/api/dashboards/home' -X 'Authorization: Bearer glsa_4ElrwSYvDrgV8GQmlyTYfsc245dMyPCE_f804430f'
+          containsSecret: true
+          match: glsa_4ElrwSYvDrgV8GQmlyTYfsc245dMyPCE_f804430f
+        - text: |
+            import requests
+
+            token = os.getenv('GRAFANA_SERVICE_ACCOUNT_TOKEN')
+            response = requests.get('https://example.grafana.net/api/dashboards/home', headers={
+                'Authorization': f'Bearer {token}',
+                'Content-Type': 'application/json'
+            })
+          containsSecret: false
+    - id: grafana-cloud-access-token
+      rspecKey: S6762
+      metadata:
+        name: Grafana Cloud access tokens should not be disclosed
+      detection:
+        pre:
+          include:
+            content:
+              - glc_eyJ
+        matching:
+          pattern: "\\b(glc_eyJ[a-zA-Z0-9]+(?:={1,2}\\B|\\b))"
```

Comment:
Same as above, I don't get why there is no / and + in the b64 detection even if I can't find more secrets when adding them. Simplification with \w is also doable here.

I'm also wondering about the \B wouldn't this miss `"glc_eyJaaa=="` for example? I know this is not a possible secret but you see what I mean.
From running your examples it seems like this is fine. There must be something I don't get with `\B`

Reply:
> Simplification with \w is also doable here.
`\w` also contains an underscore, so I'll leave this as-is.

> I'm also wondering about the \B wouldn't this miss "glc_eyJaaa==" for example
I can just leave out `\B`? Thinking about it further, there is no real need to use it here.

Reply:
My mistake for `\w`, I misread the thing. Ok with the rest.